vendor/github.com/labstack/echo/v4/middleware/jwt.go

```diff
@@ -295,7 +295,7 @@ func jwtFromHeader(header string, authScheme string) jwtExtractor {
 	return func(c echo.Context) (string, error) {
 		auth := c.Request().Header.Get(header)
 		l := len(authScheme)
-		if len(auth) > l+1 && auth[:l] == authScheme {
+		if len(auth) > l+1 && strings.EqualFold(auth[:l], authScheme) {
 			return auth[l+1:], nil
 		}
 		return "", ErrJWTMissing
```

Comment:
How come the vending changed in this patch?

Reply:
#1829 not being rebased on top of #1818

Reply:
Aha! Thanks!